### PR TITLE
Fix FB js sdk missing version

### PR DESCRIPTION
**Pull request by pior**

### CODE REVIEW

File: mtlpy/templates/base.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
             var js, fjs = d.getElementsByTagName(s)[0];
             if (d.getElementById(id)) return;
             js = d.createElement(s); js.id = id;
-            js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1";
+            js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
```
Comment (merwok) [inline mtlpy/templates/base.html]: I think HTML reserved characters still need escaping inside a script block → &amp;

Comment (pior) [inline mtlpy/templates/base.html]: It's JS string here

Comment (merwok) [inline mtlpy/templates/base.html]: But not in a CDATA section, so still HTML first.

Comment (pior) [inline mtlpy/templates/base.html]: Facebook doc :

screen shot 2017-03-12 at 10 32 35 pm

Comment (pior) [inline mtlpy/templates/base.html]: Correct answer is : https://wiki.whatwg.org/wiki/HTML_vs._XHTML#Element-specific_parsing

`script and style	In HTML, these are parsed as CDATA elements.`

Comment (merwok) [inline mtlpy/templates/base.html]: One more advantage of HTML over XML!  Thanks for checking.

(But I think it’s still required to escape `<` ?)

**Pull request merged**